dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json

```diff
@@ -78,8 +78,8 @@
         {
             "type": "Microsoft.Web/serverfarms",
             "sku": {
-                "Tier": "[parameters('sku')]",
-                "Name": "[concat(substring(parameters('sku'), 0, 1), '1')]"
+                "Tier": "[first(take(split(parameters('sku'), ' '), 1))]",
```

Comment:
I am not sure if the logic is correct here. shouldnt' it be last(split,..,)

Reply:
i tried with this and it had worked. Let me double check the docs.

Reply:
last(split()) will not work in case there are more than once spaces - not seen currently

Reply:
my intention is to split and take 2nd element in array

Reply:
changed skip to take

Reply:
sorry, changed take to skip